### DISCUSSION TITLE
Serverless v3 - How to import existing AWS API Gateway with new version

### DISCUSSION THREAD

**Discussion by mrzonca** [serverless/serverless#10456] [Q&A]
Hi, I've created conversation https://forum.serverless.com/t/serverless-v3-how-to-import-existing-aws-api-gateway-with-new-version/16465 but without answer yet. 

Can you please point me a way to handle mentioned issue?
Should I import apiGateway differently that provider.apiGateway.restApiId, if yes then how?

Or this configuration is fine but why I'm still getting error:
`When external API Gateway resource is imported via "provider.apiGateway.restApiId", property "provider.tracing.apiGateway" is ineffective.`
if there no tracing section declared, maybe it should be warning, I'm using configValidationMode: warn

**Comment by mrzonca** [comment]
Partially I found the answer, it's because serverless-plugin-datadog and x-ray tracing: true cannot run without provider.tracing.lambda: true

**Reply by pgrzesik** [reply]
Thanks for clarification - yes, if you use external plugins in might modify the configuration and it seems like that might be the case here.

**Reply by ifeltsweet** [reply]
I've submitted a PR to `serverless-plugin-datadog` to fix this issue https://github.com/DataDog/serverless-plugin-datadog/pull/262
